### PR TITLE
Bug fixes for CCC

### PR DESCRIPTION
**Pull request by jonnilundy**

### CODE REVIEW

File: web/assets/scripts/home.js [hunk 1]
```diff
@@ -7,21 +7,37 @@ if (currentUser) {
...
 	auth.signInWithRedirect(googleProvider);
 });
 
+// var btnFacebook = document.querySelector('.btn-facebook');
+
+// btnFacebook.addEventListener('click', function(e) {
+// 	var facebookProvider = new auth.provider.Facebook();
+// 	facebookProvider.setProviderScope('public_profile,email');
+// 	auth.signInWithRedirect(facebookProvider);
+// });
```
Comment (zenorocha) [inline web/assets/scripts/home.js]: Can you remove commented lines?

File: web/index.html [hunk 2]
```diff
@@ -19,6 +21,18 @@
 			</div>
 			<div class="label">Login with Google to Play</div>
 		</button>
+		<!-- <button class="btn btn-facebook" type="button">
+			<div class="brand">
+				<div class="icon icon-facebook"></div>
+			</div>
+			<div class="label">Login with Facebook to Play</div>
+		</button> -->
```
Comment (zenorocha) [inline web/index.html]: Remove comment

File: auth/container.json [hunk 3]
```diff
@@ -4,7 +4,13 @@
 	"env": {
 		"WEDEPLOY_AUTH_PASSWORD": "true",
 		"WEDEPLOY_AUTH_GOOGLE": "true",
-		"WEDEPLOY_AUTH_GOOGLE_CLIENT_ID": "199749865267-o3v9n7ps6tqtunitvgssj16fn3akfu3a.apps.googleusercontent.com",
-		"WEDEPLOY_AUTH_GOOGLE_CLIENT_SECRET": "XL9RLun1kbuMjoZVqQXoUJzo"
+		"WEDEPLOY_AUTH_GOOGLE_CLIENT_ID": "362741619553-5k705k5j7tvpp77aqsug3kg8lqlfj2kd.apps.googleusercontent.com",
+		"WEDEPLOY_AUTH_GOOGLE_CLIENT_SECRET": "b7DxIjbQ_5Mn4sn431jJ1SZ6",
+		"WEDEPLOY_AUTH_FACEBOOK": "true",
+		"WEDEPLOY_AUTH_FACEBOOK_CLIENT_ID": "154098308468413",
+		"WEDEPLOY_AUTH_FACEBOOK_CLIENT_SECRET": "20e580601b0e677ec42fb83703bf6f9f",
```
Comment (zenorocha) [inline auth/container.json]: Remove unused Facebook info.

**Comment by jonnilundy** [comment]
Should be good on the Facebook stuff.

**Comment by jonnilundy** [comment]
oops, missed one. just added commit.

**Pull request closed without merging**